spec/features/tasks/delete_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe 'Delete a task', type: :feature do
+  include_context 'when user signed in'
+
+  let(:project) { create(:project, user: current_user) }
+  let(:task) { create(:task, project: project, title: 'A test task!!!!!!!!!', user_id: current_user) }
+
+  scenario 'user try to delete a task' do
+    visit tasks_path(task)
+
+    save_and_open_page
+
+    click_on 'Destroy'
+
+    expect(page).to have_no_content('A test task!!!!!!!!!')
```

Comment:
You should also check that you've been redirected to some specific page and you can see a message stating that the task was deleted.